File: Ent_UsAdmin/EmpresaConectada.vb
Imports System.Data.OleDb

Public Class EmpresaConectada
    Private _Conectada As Boolean
    Private _ConexionEmpresaConectada As OleDbConnection
    Public Property Conectada As Boolean
        Get
            Return _Conectada

        End Get
        Set(value As Boolean)
            _Conectada = value
        End Set
    End Property
    Public Property ConexionEmpresaConectada As OleDbConnection
        Get
            Return _ConexionEmpresaConectada
        End Get
        Set(value As OleDbConnection)
            _ConexionEmpresaConectada = value
        End Set
    End Property

End Class

File: Ent_UsAdmin/Acceso/EmpresaConectada.cs
using System.Data.OleDb;

namespace ConfiaAdmin
{

    public class EmpresaConectada
    {
        private bool _Conectada;
        private OleDbConnection _ConexionEmpresaConectada;
        public bool Conectada
        {
            get
            {
                return _Conectada;

            }
            set
            {
                _Conectada = value;
            }
        }
        public OleDbConnection ConexionEmpresaConectada
        {
            get
            {
                return _ConexionEmpresaConectada;
            }
            set
            {
                _ConexionEmpresaConectada = value;
            }
        }

    }
}